main/User_config.h

```diff
@@ -175,10 +175,11 @@ int low_power_mode = DEFAULT_LOW_POWER_MODE;
 //#define ZsensorGPIOKeyCode "GPIOKeyCode" //ESP8266, Arduino, ESP32
 //#define ZsensorGPIOInput "GPIOInput" //ESP8266, Arduino, ESP32
 //#define ZmqttDiscovery "HADiscovery"//ESP8266, Arduino, ESP32, Sonoff RF Bridge
-//#define ZactuatorFASTLED "FASTLED"  //ESP8266, Arduino, ESP32, Sonoff RF Bridge
+//#define ZactuatorFASTLED "FASTLED" //ESP8266, Arduino, ESP32, Sonoff RF Bridge
 //#define ZboardM5STICKC "M5StickC"
-//#define ZboardM5STACK "ZboardM5STACK"
-//#define ZradioCC1101  "CC1101" //ESP8266, ESP32
+//#define ZboardM5STACK  "ZboardM5STACK"
+//#define ZradioCC1101   "CC1101"   //ESP8266, ESP32
+//#define ZactuatorPWM   "PWM"      //ESP32
```

Comment:
Could you add also please the module into [esp32dev-all environment](https://github.com/1technophile/OpenMQTTGateway/blob/605da549b94be482e0d0e3abfff97ff2e3bb5cb7/platformio.ini#L169) so as to enable the CI to build the code.

Reply:
Done - but I had to make the following changes locally to make it build....

```
-rfm69 = RFM69@4a6f77ad47
+rfm69 = RFM69

-bmp180 = BMP180@efac46bd8d
+bmp180 = BMP180

-htu21 = SparkFun HTU21D Humidity and Temperature Sensor Br@1.1.3
+htu21 = SparkFun HTU21D Humidity and Temperature Sensor Breakout@1.1.3
```